draft-wang-ppm-dap-taskprov.md

```diff
@@ -278,17 +281,13 @@ differential privacy (DP). The opaque `dp_config` contains the following structu
 enum {
     reserved(0), /* Reserved for testing purposes */
     none(1),
-    aggregator_discrete_gaussian(5),
```

Comment:
Don't think this is standardized yet? @cjpatton

Reply:
You're right, I'm not sure how this got added. It looks like I did somehow 8f806955